.formatter.exs

```diff
@@ -1,5 +1,4 @@
 # Used by "mix format"
 [
-  inputs: ["{mix,.formatter}.exs", "{config,lib,test,patches}/**/*.{ex,exs}"],
-  subdirectories: ["examples/*"]
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test,patches}/**/*.{ex,exs}"]
```

Comment:
Why this change?

Reply:
Otherwise mix format also formats examples, and the liveview example has phoenix in the formatter deps and formatting fails since phoenix is not a dependency of popcorn

Reply:
Here's what happens without this change: https://github.com/software-mansion/popcorn/actions/runs/19166714172/job/54789030104